### DISCUSSION TITLE
24h time format?

### DISCUSSION THREAD

**Discussion by trexxeon** [babybuddy/babybuddy#879] [Q&A]
Hello, 

How do I enable 24h time format?

My country doesn't use the am/pm system so I'd prefer the 24h time format.

**Comment by shalak** [comment]
DId you happen do find the solution?

As per [this PR](https://github.com/babybuddy/babybuddy/pull/150), I set `USE_24_HOUR_TIME_FORMAT: true` in my docker compose, but did not solve the issue...


I'm also interested in setting up the ISO date format (YYYY-MM-DD, instead of the MM/DD/YYYY)

**Reply by trexxeon** [reply]
Hey, didn’t find a solution. Gave up and moved on

**Comment by cdubz** [comment]
Unfortunately, that option is removed from v2 onwards. There were a lot of frustrating little bugs with overriding the default locale behaviors at runtime. Baby Buddy now relies on [Django's locale configurations](https://github.com/django/django/tree/stable/5.1.x/django/conf/locale) for time display.

I'd be happy to bring this feature back, but have not had time myself to work on it.

**Reply by shalak** [reply]
How can I tune django's locale configuration for baby buddy?

**Reply by cdubz** [reply]
Hmmm, while it may be possible to patch Django as a dependency, I'm not sure how you'd go about it. It would depend on how you are managing your Baby Buddy deployment (and may or may not be possible with whatever solution you are using).

I'll try to look in to getting this improved directly in Baby Buddy sooner than later. As the kids become older... the time to work on stuff like this really diminishes 😁 

You can watch babybuddy/issues/679 for movement on the issue.
